lms/djangoapps/instructor/views/api.py

```diff
@@ -2295,10 +2296,9 @@ def list_report_downloads(_request, course_id):
     """
     course_id = SlashSeparatedCourseKey.from_deprecated_string(course_id)
     report_store = ReportStore.from_config(config_name='GRADES_DOWNLOAD')
-
     response_payload = {
         'downloads': [
-            dict(name=name, url=url, link='<a href="{}">{}</a>'.format(url, name))
+            dict(name=name, url=url, link=HTML('<a href="{}">{}</a>').format(url, name))
```

Comment:
The problem is coming from this line: https://github.com/edx/edx-platform/blob/master/common/djangoapps/util/json_request.py#L69

```
> /edx/app/edxapp/edx-platform/common/djangoapps/util/json_request.py(69)__init__()
-> content = json.dumps(resp_obj, cls=encoder, indent=2, ensure_ascii=False)
(Pdb) n
> /edx/app/edxapp/edx-platform/common/djangoapps/util/json_request.py(70)__init__()
-> kwargs.setdefault("content_type", "application/json")
(Pdb) content
u'{\n  "downloads": [\n    {\n      "url": "https://1.mock.url", \n      "link": &#34;<a href=\\&#34;https://1.mock.url\\&#34;>mock_file_name_1</a>&#34;, \n      "name": "mock_file_name_1"\n    }, \n    {\n      "url": "https://2.mock.url", \n      "link": &#34;<a href=\\&#34;https://2.mock.url\\&#34;>mock_file_name_2</a>&#34;, \n      "name": "mock_file_name_2"\n    }\n  ]\n}'
(Pdb) resp_obj
{'downloads': [{'url': 'https://1.mock.url', 'link': Markup(u'<a href="https://1.mock.url">mock_file_name_1</a>'), 'name': 'mock_file_name_1'}, {'url': 'https://2.mock.url', 'link': Markup(u'<a href="https://2.mock.url">mock_file_name_2</a>'), 'name': 'mock_file_name_2'}]}
```